Dockerfile

```diff
@@ -3,13 +3,13 @@ FROM postgres:11.2-alpine
 ARG BUILD_DATE
 ARG SOURCE_COMMIT
 
-LABEL maintainer "EGA System Developers"
+LABEL maintainer "NeIC System Developers"
 LABEL org.label-schema.schema-version="1.0"
 LABEL org.label-schema.build-date=$BUILD_DATE
-LABEL org.label-schema.vcs-url="https://github.com/EGA-archive/LocalEGA-db"
+LABEL org.label-schema.vcs-url="https://github.com/neicnordic/LocalEGA-db"
 LABEL org.label-schema.vcs-ref=$SOURCE_COMMIT
 
-ENV SSL_SUBJ             /C=ES/ST=Spain/L=Barcelona/O=CRG/OU=SysDevs/CN=LocalEGA/emailAddress=dev.ega@crg.eu
+ENV SSL_SUBJ             /C=SE/ST=Sweden/L=Uppsala/O=NBIS/OU=SysDevs/CN=LocalEGA
 ENV TZ                   Europe/Madrid
```

Comment:
Should the timezone be changed as well (_I_ would prefer Europe/Stockholm, but other people might think otherwise).

Reply:
would make sense to have it `Europe/Stockholm` as the SSL_SUBJ are related to Sweden. `UTC` could be used as an alternative.